README.md

```diff
@@ -153,7 +153,7 @@ Prometheus or a StatsD interface supporting data store for metrics. This is requ
 ## Protos
 Data is encapsulated into protobuf messages of different types. We do not recommend making changes, but if you need to recompile them you can always do so with:
 
-  1. Install protoc, currently on version 3.21.12: https://grpc.io/docs/protoc-installation/
+  1. Install protoc, currently on version 25.0: https://grpc.io/docs/protoc-installation/
```

Comment:
should this be `4.25.0`?

Reply:
Updated. The download URL starting version 4 became a little different `https://github.com/protocolbuffers/protobuf/releases/download/v25.0/protoc-25.0-linux-x86_64.zip` hence I got confused